src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -5,8 +5,19 @@
 
 using System;
```

Comment:
Just a suggestion, in next visit of these files, could we arrange the namespaces alphabetically starting with 's'.